sem_covid/services/store_registry.py

```diff
@@ -70,7 +70,7 @@ def es_index_store(self) -> IndexStoreABC:
         """
         es_pandas_client = es_pandas(config.ELASTICSEARCH_HOST_NAME,
                                      http_auth=(config.ELASTICSEARCH_USERNAME, config.ELASTICSEARCH_PASSWORD),
-                                     port=config.ELASTICSEARCH_PORT, http_compress=True)
+                                     port=config.ELASTICSEARCH_PORT, http_compress=True, timeout=3600)
```

Comment:
Let's make this TIMEOUT a constant or a config variable. 

It can a module-level constant for now. Why this parameter seems to have a big impact on the outcome provided that the server has an unstable internet connection.